endersgame/crunch/replay.py

```diff
@@ -52,14 +54,24 @@ def filter(self, pt: MarketDataPoint):
             return pt.securityId in self.substream_ids_only
         return True
 
-    def tick(self) -> StreamBatch:
+    def values(self, delay: float) -> Iterator[StreamPoint]:
+        while self.idx < len(self.timestamps):
+            batch = self.tick()
+            for pt in batch.points:
+                if pt.substream_id not in self.substream_ids_only:
+                    continue
+                yield pt
+                time.sleep(delay)
```

Comment:
todo: replay should not sleep and just run as fast as possible

Or enable sleeping with a flag.